L1Trigger/L1TGlobal/plugins/L1TGlobalProducer.cc

```diff
@@ -403,6 +401,16 @@ void L1TGlobalProducer::produce(edm::Event& iEvent, const edm::EventSetup& evSet
       m_l1GtMenu->print(std::cout, printV);
 
     m_l1GtMenuCacheID = l1GtMenuCacheID;
+
+    //for getting model version to condition class via GlobalBoard.runGTL, comes from menu rather than config
+    if (gtParser.gtTriggerMenuName() == "L1Menu_Collisions2024_v1_0_0") {
```

Comment:
if one tries to use a menu even older than `L1Menu_Collisions2024_v1_0_0` this would not throw an exception but some stack trace, right? Is it what we want here?

Reply:
`L1Menu_Collisions2024_v0_0_0` does.

Reply:
@eyigitba @quinnanm please fix this.

Reply:
ok, then @quinnanm can add that to here as well.

Reply:
apologies, I wasn't aware of menu v0_0_0. I added that in the most recent commit [69cebf8](https://github.com/cms-sw/cmssw/pull/44054/commits/69cebf8efdde908784a2369a3fe733fe09635576) so that the exception thrown is now more informative (reminds user to use L1Menu_Collisions2024_v1_0_1 or greater rather than AnomalyDetectionTrigger_i426 not found in condition map)